sfputil/main.py

```diff
@@ -1333,6 +1333,8 @@ def download_firmware(port_name, filepath):
     with click.progressbar(length=file_size, label="Downloading ...") as bar:
         address = 0
         BLOCK_SIZE = MAX_LPL_FIRMWARE_BLOCK_SIZE if lplonly_flag else maxblocksize
+        if maxblocksize < BLOCK_SIZE:
+            BLOCK_SIZE = maxblocksize
```

Comment:
@stephenxs can you revise the code as follows: 
```
if (lpl page) {
  // At max the driver allows 116 Bytes write for LPL
   block size = min (maxblocksize , MAX_LPL_SIZE)
} else {
   block_size = maxblocksize
}
 ```

Reply:
@stephenxs its python, so hope you understand :)

Reply:
@prgeor 
LGTM. Done.